Assets/GothicVR/Scripts/Player/Menu/MenuDropdownController.cs

```diff
@@ -1,134 +1,60 @@
 using System.Collections.Generic;
 using TMPro;
 using UnityEngine;
-using GVR.Phoenix.Interface;
 using GVR.Manager;
 using GVR.Util;
 
 namespace GVR.Player.Menu
 {
     public class WorldSelectorDropdownController : SingletonBehaviour<WorldSelectorDropdownController>
     {
-        private Dictionary<string, string> worlds = new();
-        [SerializeField] private TMP_Dropdown worldDropdown;
-
-        private Dictionary<string, List<string>> waypoints = new();
+        private List<string> waypoints = new();
         [SerializeField] private TMP_Dropdown waypointDropdown;
 
         private void Start()
         {
-            SetWorldDropdown();
             SetWaypointDropdown();
-
-            worldDropdown.value = 0;
         }
 
-        private void SetWorldDropdown()
+        void SetWaypointDropdown()
         {
-            worldDropdown.options.Clear();
 
-            worlds.Add("World", "world.zen");
-            worlds.Add("Old Mine", "oldmine.zen");
-            worlds.Add("Orc Graveyard", "orcgraveyard.zen");
-            worlds.Add("Orc Temple", "orctempel.zen");
-            worlds.Add("Free Mine", "freemine.zen");
-
-            worldDropdown.itemText.font = GameData.I.EmptyFont;
-            worldDropdown.onValueChanged.AddListener(WorldDropdownItemSelected);
-            foreach (var item in worlds)
+            waypoints = new()
+            {
+                "START",
+                "ENTRANCE_SURFACE_OLDMINE",
+                "ENTRANCE_SURFACE_ORCGRAVEYARD",
+                "ENTRANCE_SURFACE_ORCTEMPLE",
+                "ENTRANCE_FREEMINECAMP_FREEMINE",
+                "OCC_CHAPEL_UPSTAIRS",
+                "NC_KDW_CAVE_CENTER",
+                "PSI_TEMPLE_COURT_GURU",
+                "DT_E2_06",
+            };
```

Comment:
You could also think of having this list data created inside scene view. No need to have it here inside code, right?